README-fr.md

```diff
@@ -99,57 +99,57 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
     * [Étape 1: Regarder la vidéo de mise à l'échelle](#step-1-review-the-scalability-video-lecture)
     * [Étape 2: Lire l'article sur la mise à l'échelle](#step-2-review-the-scalability-article)
     * [Étapes suivantes](#next-steps)
-* [Performance vs scalability](#performance-vs-scalability)
-* [Latency vs throughput](#latency-vs-throughput)
-* [Availability vs consistency](#availability-vs-consistency)
-    * [CAP theorem](#cap-theorem)
-        * [CP - consistency and partition tolerance](#cp---consistency-and-partition-tolerance)
-        * [AP - availability and partition tolerance](#ap---availability-and-partition-tolerance)
-* [Consistency patterns](#consistency-patterns)
-    * [Weak consistency](#weak-consistency)
-    * [Eventual consistency](#eventual-consistency)
-    * [Strong consistency](#strong-consistency)
-* [Availability patterns](#availability-patterns)
-    * [Fail-over](#fail-over)
-    * [Replication](#replication)
-* [Domain name system](#domain-name-system)
-* [Content delivery network](#content-delivery-network)
-    * [Push CDNs](#push-cdns)
-    * [Pull CDNs](#pull-cdns)
-* [Load balancer](#load-balancer)
-    * [Active-passive](#active-passive)
-    * [Active-active](#active-active)
-    * [Layer 4 load balancing](#layer-4-load-balancing)
-    * [Layer 7 load balancing](#layer-7-load-balancing)
-    * [Horizontal scaling](#horizontal-scaling)
-* [Reverse proxy (web server)](#reverse-proxy-web-server)
-    * [Load balancer vs reverse proxy](#load-balancer-vs-reverse-proxy)
-* [Application layer](#application-layer)
+* [Performance vs évolutivité](#performance-vs-scalability)
+* [Latence vs haut débit](#latency-vs-throughput)
+* [Disponibilité vs Cohérence](#availability-vs-consistency)
+    * [Théorème CAP ou CDP](#cap-theorem)
+        * [CP - Cohérence et tolérance au Partitionnement](#cp---consistency-and-partition-tolerance)
+        * [DP - Disponibilité et tolérance au Partitionnement](#ap---availability-and-partition-tolerance)
+* [Patrons de Cohérence](#consistency-patterns)
+    * [Faible cohérence](#weak-consistency)
+    * [Cohérence](#eventual-consistency)
+    * [Cohérence Forte](#strong-consistency)
+* [Patrons de Disponibilité](#availability-patterns)
+    * [Basculement](#fail-over)
+    * [Réplication](#replication)
+* [Système de Noms de Domaine](#domain-name-system)
+* [Réseau de diffusion de contenu](#content-delivery-network)
+    * [Injection RDC (CDNs)](#push-cdns)
+    * [Récupération RDC (CDNs)](#pull-cdns)
+* [Répartiteur de Charge](#load-balancer)
+    * [Actif-passif](#active-passive)
+    * [Actif-actif](#active-active)
+    * [Répartition de charge couche 4](#layer-4-load-balancing)
+    * [Répartition de charge couche 7](#layer-7-load-balancing)
+    * [Mise à l'échelle horizontale](#horizontal-scaling)
+* [Proxy inverse (serveur web)](#reverse-proxy-web-server)
+    * [Répartiteur de Charge vs Proxy inverse](#load-balancer-vs-reverse-proxy)

```

Comment:
```suggestion
    * [Répartiteur de charge vs Proxy inverse](#load-balancer-vs-reverse-proxy)
```

Reply:
Fixed in another commit https://github.com/spuyet/system-design-primer/pull/1/commits/537031afc5a4b2774e74e7a84a35f6a28a043b55
 as I can't edit in this viewer.